docs/en/query_language/select.md

```diff
@@ -564,29 +564,37 @@ ClickHouse doesn't directly support syntax with commas, so we don't recommend us
 
 Tables for `ASOF JOIN` must have an ordered sequence column. This column cannot be alone in a table, and should be one of the data types: `UInt32`, `UInt64`, `Float32`, `Float64`, `Date`, and `DateTime`.
 
-You can use the following types of syntax:
+Syntax `ASOF JOIN ... ON`:
 
-- `ASOF JOIN ... ON`
+```sql
+SELECT expressions_list
+FROM table_1
+ASOF LEFT JOIN table_2
+ON equi_cond AND closest_match_cond
+```
+
+You can use any number of equality conditions and exactly one closest match condition. For example, `SELECT count() FROM A ASOF LEFT JOIN B ON A.a == B.b AND B.t <= A.t`. 
 
-    ```sql
-    SELECT expressions_list
-    FROM table_1
-    ASOF LEFT JOIN table_2
-    ON equi_cond AND closest_match_cond
-    ```
+Conditions supported for the closest match:
 
-    You can use any number of equality conditions and exactly one closest match condition. For example, `SELECT count() FROM A ASOF LEFT JOIN B ON A.a == B.b AND B.t <= A.t`. Only `table_2.some_col <= table_1.some_col` and `table_1.some_col >= table2.some_col` condition types are available. You can't apply other conditions like `>` or `!=`.
+- `table_1.some_col > table_2.some_col`
+- `table_1.some_col < table_2.some_col`
+- `table_1.some_col >= table_2.some_col`
+- `table_1.some_col <= table_2.some_col`
+- `table_2.some_col <= table_1.some_col`
```

Comment:
- In previous version < and > are strictly forbidden
- I don't think we should explicitly write all possible conditions. Only one version with table2 <= table1 makes it more confusing

Reply:
I think we could say here: You are able to use one of these conditions `>`, `>=`, `<`, `<=`